modules/graphql_content/src/Plugin/GraphQL/Enums/Languages.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Drupal\graphql_content\Plugin\GraphQL\Enums;
+
+use Drupal\Core\Language\LanguageManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\graphql_core\GraphQL\EnumPluginBase;
+use Drupal\graphql_core\GraphQLSchemaManagerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Generates an enumeration of numbers.
+ *
+ * @GraphQLEnum(
+ *   id = "language",
+ *   name = "Language"
```

Comment:
This might actually be why it's failing... There is already a type named "Language".